docs/specs/alcproxies.md

```diff
@@ -0,0 +1,251 @@
+# Proxy Creation for AssemblyLoadContext to Replace AppDomain
+## Problem
+LoadContexts currently work as containers for assemblies, to work as a semi-replacement for AppDomain. However, the current implementation of AssemblyLoadContext creates a Type-isolation boundary between an Application and context, compared to the Object-isolation boundary that AppDomain created. This creates complications for developers intending on using AssemblyLoadContext in similar ways to AppDomain.
+
+In the past, a common use of AppDomain was to load an assembly into a separate execution environment, where developers were able to load objects and use them as if they were in the original domain. 
+
+
+## Comparison
+
+### AppDomain
+For AppDomains, developers could load objects and execute methods like so (Shortened version from [here](https://docs.microsoft.com/en-us/dotnet/api/system.appdomain?view=netframework-4.8 )):
```

Comment:
Update the URL: https://docs.microsoft.com/dotnet/api/system.appdomain?view=netframework-4.8

The `en-us` should be removed so the browser viewing the page can apply the users locale preference.